examples/l3-resources/atlas-bootstrap.ts

```diff
@@ -0,0 +1,47 @@
+// Copyright 2023 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as cdk from "aws-cdk-lib";
+import { Construct } from 'constructs';
+import {
+  MongoAtlasBootstrap,
+  MongoAtlasBootstrapProps,
+  AtlasBasicResources,
+} from "awscdk-resources-mongodbatlas";
+
+export class AtlasBoostrapExample extends cdk.Stack {
+    constructor(scope: Construct, id: string, props?: cdk.StackProps) {
+      super(scope, id, props);
+  
+    const mongoDBProfile = "development";
+    const roleName = "MongoDB-Atlas-CDK-Excecution";
+
+    const publicKey = new cdk.CfnParameter(this, "atlasPublicKey", {
+    noEcho: true,
+    type: "String",
+    });
+    const privateKey = new cdk.CfnParameter(this, "atlasPrivateKey", {
+    noEcho: true,
+    type: "String",
+    });
```

Comment:
I don't think we should allow user to pass private key through the parameter. This means people with cloudformation stack describe privilege will be able to see the plaintext of the private key. If you deploy this and go to the cloudformation console, you should be able to see all the key values. Maybe we should avoid that.

Reply:
this are gone, we dont need them anymore, ill remove them